message_sender/tests.py

```diff
@@ -4007,7 +4007,7 @@ def test_send_text_to_send_hsm(self):
         send_text should delegate to send_hsm when there are HSM config values.
         """
         sender = WhatsAppApiSender(
-            "http://whatsapp", "test-token", "hsm-namespace", "hsm-element-name"
+            "http://whatsapp", "test-token", "hsm-namespace", "hsm-element-name", "ttl"
```

Comment:
Instead of using the string `ttl`, this should rather be the value that we want the ttl to be. We should then also assert that the request made to the whatsapp API includes the value of `ttl` that we've put here.

Reply:
so e.g. use a value of 604800 and assert that?